examples/todomvc/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "todomvc",
+  "version": "1.0.0",
+  "description": "[![Demo](https://img.shields.io/website?label=demo&url=https%3A%2F%2Fexamples.yew.rs%2Ftodomvc)](https://examples.yew.rs/todomvc)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "cypress": "cypress open",
+    "cypress:run": "cypress run --browser chrome"
+  },
+  "author": "",
+  "license": "ISC",
```

```suggestion
  "license": "(MIT OR Apache-2.0)",
```

Yew is licensed in MIT OR Apache-2.0 at the users choice, I believe tests should also follow this license.

---

Thanks for the review, I fixed it to MIT OR Apache-2.0